exam001/models/DevCell.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  DevCell.swift
+//  exam001
+//
+//  Created by Seif Ghotouk on 11/13/18.
+//  Copyright © 2018 Adnan Al-Akhrass. All rights reserved.
+//
+
+import UIKit
+
+class DevCell: UITableViewCell {
+    
+    public var cellIndex = 0
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
```

Comment:
@ADIGA-Cheezo 
Remove auto generated comments, comment should not be used at all except:
1. When defining `marks` such as `// MARK: - Properties`
2. Describing block of code that is should be cleared out, also this should be very rare to happen.